cmd/openshift-install/destroy.go

```diff
@@ -44,26 +44,40 @@ func newDestroyCmd() *cobra.Command {
 }
 
 func newDestroyClusterCmd() *cobra.Command {
-	return &cobra.Command{
+	destroyCmd := &cobra.Command{
 		Use:   "cluster",
 		Short: "Destroy an OpenShift cluster",
 		Args:  cobra.ExactArgs(0),
-		Run: func(_ *cobra.Command, _ []string) {
+		Run: func(cmd *cobra.Command, _ []string) {
 			cleanup := command.SetupFileHook(command.RootOpts.Dir)
 			defer cleanup()
+			destroyVolumes, err := cmd.Flags().GetBool("destroy-persistent-volumes")
+			if err != nil {
+				logrus.Fatal(err)
+			}
+			kubeConfig, err := cmd.Flags().GetString("kubeconfig")
+			if err != nil {
+				logrus.Fatal(err)
+			}
 
-			err := runDestroyCmd(command.RootOpts.Dir, os.Getenv("OPENSHIFT_INSTALL_REPORT_QUOTA_FOOTPRINT") == "true")
+			err = runDestroyCmd(command.RootOpts.Dir, os.Getenv("OPENSHIFT_INSTALL_REPORT_QUOTA_FOOTPRINT") == "true", destroyVolumes, kubeConfig)
 			if err != nil {
 				logrus.Fatal(err)
 			}
 			logrus.Infof("Uninstallation complete!")
 		},
 	}
+
+	destroyCmd.Flags().Bool("destroy-persistent-volumes", false, "cordons and drains nodes, deletes all persistent volumes.")
+	destroyCmd.Flags().String("kubeconfig", "", "path to kubeconfig to allow the installer to destroy persistent volumes.")
```

Comment:
Perhaps we could get from the `KUBECONFIG` environment variable too? or is it intentionally a flag to prevent destructive stuffs?

Reply:
I suppose if KUBECONFIG env var was set that could be our pairing indicator that its ok to proceed.